### ISSUE TITLE
I'll update Android

### ISSUE THREAD

**Issue by Sternbach-Software** [sputt/wabbitemu#33]
I am an Android dev and would like to update the app so that it can go on the play store again. Just send me the source and I will take care of it.

**Comment by gerroon** [comment]
> I am an Android dev and would like to update the app so that it can go on the play store again. Just send me the source and I will take care of it.

Do you have a release date for this?

**Comment by shmueldabomb441** [comment]
I was never sent the source code.

**Comment by milnak** [comment]
@shmueldabomb441 source link is [here](https://github.com/sputt/wabbitemu/issues/32#issuecomment-1085942979)

**Comment by shmueldabomb441** [comment]
@gerroon is it urgent?

**Comment by gerroon** [comment]
> @gerroon is it urgent?
Not urgent:) I was looking to see if there was n update.

**Comment by JamesOsborn-SE** [comment]
would be great if we can get it on f-droid: https://f-droid.org I'm willing to help

**Comment by levivk** [comment]
@Sternbach-Software Are you still planning to do this?

**Comment by Sternbach-Software** [comment]
@levivk Yes. It has resurfaced in my mind several times and I have mentally chipped away at it in terms of preparing ahead of time for what is necessary. Let me take a swing at it now.

**Comment by Sternbach-Software** [comment]
You can follow my updates/activity by looking at "history" or "commits" in my forked repo I linked above.

**Comment by Sternbach-Software** [comment]
https://github.com/sputt/wabbitemu/issues/32#issuecomment-1367829738

**Comment by Sternbach-Software** [comment]
How upset would people be if I resigned the app to use regular [Material Design 2](https://m2.material.io/components/buttons) or [3](https://m3.material.io/components/buttons/overview) buttons, instead of overlaying buttons on TI calculator images? This would make the app
- easier to maintain and updates come sooner
- less error prone and solve the existing issues with button touch zones being in the wrong place, etc.
- a smoother user experience
- less "familiar"/nostalgic for the hardcore TI fans.

**Comment by UndarkAido** [comment]
For me, the whole point of Wabitemu is that you can pick it up and all of your experience and muscle memory from the physical calculator is instantly transferrable to the emulator. If it wasn't I'd just use a native Android graphing calculator app.

**Comment by Sternbach-Software** [comment]
@UndarkAido Can you do an experiment for me? Which calculator are most familiar with? It would be best if you said TI-84/Plus/SE

**Comment by levivk** [comment]
I'm inclined to agree with @UndarkAido. I use Wabbitemu over a native android calculator because I want something that looks and feels like something I'm already familiar with. Also, I think that not overlaying a TI calculator image would create a less enjoyable experience.

**Comment by UndarkAido** [comment]
I use a TI-84 Plus C Silver Edition, which I don't seem to be able to generate a profile for on my new phone. Darn.

**Comment by DanHolli** [comment]
I'm very familiar with the TI84 plus SE but also pretty familiar with the TI 83 as well, let me know if you need someone to help test

**Comment by Sternbach-Software** [comment]
@DanHolli123 Definitely. Once I am done the prototype, I will ask you to test whether you can use it easily.

**Comment by AerospaceDoe** [comment]
Hi @Sternbach-Software, could you provide a copy of your built APK file?
I tried to build from [your repository](https://github.com/Sternbach-Software/Wabbitemu-Android) but cannot see the ROM file in the app.
It seems to be a storage permission issue.

**Comment by Sternbach-Software** [comment]
@AerospaceDoe I'm actually currently debugging that.
